# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1470634229 -7200
#      Mon Aug 08 07:30:29 2016 +0200
# Node ID 6f1887b237d559c68cb381c37805eb4bc92b2fbd
# Parent  6bb0f32d22a55ff23a3e9e866becc3b68d700a90
compare_plot_demos.m: Fix default path on Windows (Bug #48717)

diff --git a/scripts/testfun/private/compare_plot_demos.m b/scripts/testfun/private/compare_plot_demos.m
--- a/scripts/testfun/private/compare_plot_demos.m
+++ b/scripts/testfun/private/compare_plot_demos.m
@@ -41,17 +41,18 @@
 ##
 ## @end deftypefn
 
 ## Author: Ben Abbott  <bpabbott@mac.com>
 
 function compare_plot_demos (varargin)
 
   arg.toolkits = available_graphics_toolkits ();
-  arg.directories = {"plot/appearance", "plot/draw", "plot/util", "image"};
+  arg.directories = fullfile ("plot", {"appearance", "draw", "util"});
+  arg.directories(end + 1) = "image";
   arg.fmt = "png";
   arg.fcn_file = "dump_plot_demos.m";
   arg.replace_images = false;
 
   for n = 1:2:numel(varargin)
     if (! ischar (varargin{n}))
       print_usage ();
     else
